pythonAPI/germanetpy_tutorial.ipynb

```diff
@@ -423,38 +445,96 @@
     "leber_trans = germanet.get_synset_by_id(\"s83979\")\n",
     "relatedness_calculator = PathBasedRelatedness(germanet=germanet, category=WordCategory.nomen, max_len=35,\n",
-    "                                             max_depth=20, synset_pair=(johannis_wurm, leber_trans))\n",
+    "                                              max_depth=20, synset_pair=(johannis_wurm, leber_trans))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now use the `relatedness_calculator` object to find out whether *Trompete* (trumpet) is more closely related to *Posaune* (trombone) or to *Flöte* (flute):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "trompete = germanet.get_synsets_by_orthform(\"Trompete\").pop()\n",
     "flöte = germanet.get_synsets_by_orthform(\"Flöte\").pop()\n",
     "posaune = germanet.get_synsets_by_orthform(\"Posaune\").pop()\n",
-    "Klavier = germanet.get_synsets_by_orthform(\"Klavier\").pop()\n",
-    "Flügel_synsets = germanet.get_synsets_by_orthform(\"Flügel\")\n",
     "trompete_posaune = relatedness_calculator.simple_path(trompete, posaune)\n",
     "trompete_flöte = relatedness_calculator.simple_path(trompete, flöte)\n",
     "\n",
-    "print(\"Based on the simple path measure, is Trompete more similar to Posaune, than to Flöte? %s\" % str(trompete_posaune > trompete_flöte))\n",
+    "trompete_posaune > trompete_flöte "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Path based relatedness measures can also be used to disambiguate word senses. This example shows how to find the sense of *Flügel* (wing, blade, grand) which is most similar to *Klavier* (piano) according to three different path-based measures:. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Klavier = germanet.get_synsets_by_orthform(\"Klavier\").pop()\n",
+    "Flügel_synsets = germanet.get_synsets_by_orthform(\"Flügel\")\n",
+    "\n",
+    "results = []\n",
     "highest_sim_simple = 0.0\n",
     "highest_sim_leacock = 0.0\n",
     "highest_sim_wu = 0.0\n",
     "most_similar_synset = None\n",
+    "\n",
     "for synset in Flügel_synsets:\n",
     "    if synset.word_category == WordCategory.nomen:\n",
     "        sim_simple = relatedness_calculator.simple_path(synset, Klavier, normalize=True)\n",
     "        sim_leacock = relatedness_calculator.leacock_chodorow(synset, Klavier, normalize=True)\n",
     "        sim_wu = relatedness_calculator.wu_and_palmer(synset, Klavier, normalize=True)\n",
-    "        print(\"\\n These are the similarities between the synset for Klavier and %s : \\n Simple Path : %.2f\\n Leackock and Chodorow: %.2f\\n Wu and Palmer : %.2f\" % (str(synset), sim_simple, sim_leacock, sim_wu));\n",
+    "        results.append([synset.id, sim_simple, sim_leacock, sim_wu])\n",
```

Comment:
Habe mich gefragt ob es vielleicht sinnvoller ist die lexical units, statt die id anzuzeigen weil man dann auch weiß wie die similarity zu welchem sense ist, da man ja mit der id erstmal nix anfangen kann wenn es nicht nachschaut. oder meinst du das wäre dann zu lang um in der Tabelle angezeigt zu werden?

Reply:
Es wäre nicht zu lang. In diesem Fall sind die Lexunits auch nicht super hilfreich, weil mehrere nur das einzige Wort "Flügel" enthalten. Sieht so aus:

 
![Screenshot 2020-04-29 at 12 58 47](https://user-images.githubusercontent.com/4525191/80590098-9febc400-8a1b-11ea-92e7-1f69050c5a74.png)

Es schadet ohnehin nichts, die Lexunits einzuschließen.

Reply:
ja immerhin hat man für 4 sense noch eine info darüber was gemeint ist.